src/utils.ts

```diff
@@ -19,3 +32,22 @@ export function isDir (path: string | URL): boolean {
     return false
   }
 }
+
+export function isFile (path: string | URL): boolean {
```

Comment:
How should we treat symlinks?

Whatever decision you pick, can you document it in English here in a comment above this function?

Reply:
I doubt symlinks should be accepted as valid keys in a key-value store, especially since they would likely point to somewhere outside the data folder. So I'd rather skip them when migrating from `fs`.
Besides, if a symlink gets passed in as command argument e.g. `--out`, I'm not sure but I guess the link should be followed?

Reply:
Yeah if the output directory includes a symlink we should follow it